### DISCUSSION TITLE
Real World Data Science - Contributors: check out the new Real World Data Science template repo on GitHub

### DISCUSSION THREAD

**Discussion by giscus** [realworlddatascience/realworlddatascience.github.io#95] [General]
# Real World Data Science - Contributors: check out the new Real World Data Science template repo on GitHub

The ‘RWDS_post_template’ repository, created by Finn-Ole Höner, demonstrates various Quarto features and helps site contributors to create content using our house style and format.

https://realworlddatascience.net/viewpoints/editors-blog/posts/2023/09/12/template.html

**Comment by finnoh** [comment]
It was a pleasure to help!
